include/seqan3/io/detail/ignore_output_iterator.hpp

```diff
@@ -42,16 +42,16 @@ class ignore_output_iterator
     using iterator_category = std::output_iterator_tag;
     //!\}
 
-    /*!\name Constructor, destructor and assignment
+    /*!\name Constructors, destructor and assignment
+     * \brief Rule of five explicitly defaulted.
```

Comment:
You deleted this comment in the other cases so I would not introduce this line here